content/source/docs/cloud/vcs/bitbucket-server.html.md

```diff
@@ -59,14 +59,14 @@ Leave the page open in a browser tab, and remain logged in as an admin user.
 
 1. Enter the URL of your Bitbucket Server instance in the **HTTP URL** and **API URL** fields.
 
-    If your Bitbucket Server instance does not have a [context path](https://confluence.atlassian.com/bitbucketserver/moving-bitbucket-server-to-a-different-context-path-776640153.html) set, the **API URL** should be the same as the **HTTP URL**.
+-> **Context Path**  If your Bitbucket Server instance does not have a [context path](https://confluence.atlassian.com/bitbucketserver/moving-bitbucket-server-to-a-different-context-path-776640153.html) set, the **API URL** should be the same as the **HTTP URL**.
```

Comment:
By removing the leading whitespace, you are breaking the numbering of the ordered lists.

Reply:
```suggestion
    ~> **Note:** The values for the **HTTP URL** and **API URL** fields differ depending on whether or not your Bitbucket Server instance has a [context path](https://confluence.atlassian.com/bitbucketserver/moving-bitbucket-server-to-a-different-context-path-776640153.html) set.
    
    If your Bitbucket Server instance does not have a context path set, the **API URL** should be the same as the **HTTP URL**.
```

Reply:
Thank you for catching the indent thing, I was just about to start untangling that when I noticed you got it. :D Tweaking the flag here (yellow boxes get "important" instead of "note"):

```suggestion
    ~> **Important:** The values for the **HTTP URL** and **API URL** fields differ depending on whether or not your Bitbucket Server instance has a [context path](https://confluence.atlassian.com/bitbucketserver/moving-bitbucket-server-to-a-different-context-path-776640153.html) set.
    
    If your Bitbucket Server instance does not have a context path set, the **API URL** should be the same as the **HTTP URL**.
```